app/controllers/concerns/hyrax/collections_controller_behavior.rb

```diff
@@ -106,6 +107,13 @@ def after_create_error
     end
 
     def create
+      # Manual load and authorize necessary because Cancan will pass in all
+      # form attributes. When `permissions_attributes` are present the
+      # collection is saved without a value for `has_model.`
+      @collection = ::Collection.new
```

Comment:
If we have these lines here, we should not have `load_and_authorize_resource :create`